common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -46,6 +46,11 @@ class InheritanceMixin(XBlockMixin):
         help=_("Enter the default date by which problems are due."),
         scope=Scope.settings,
     )
+    due_num_weeks = Integer(
+        display_name=_("Number of Weeks Due By"),
+        help=_("Enter the number of weeks the problems are due by"),
```

Comment:
We should make a note to improve this help messaging once we chat with Hilary/Kaitlin. I think it would be good for example to call out that it is shifted based on the Schedule Start Date, but not sure on exact phrasing. Although this is internal only I believe so maybe we don't need ass much input from Kaitlin/Hilary as they are more focused on external facing language.